Android.mk

```diff
@@ -16,6 +16,6 @@
 
 LOCAL_PATH := $(call my-dir)
 
-ifneq ($(filter beyond0lte, $(TARGET_DEVICE)),)
+ifneq ($(filter beyond0lte,beyond2lte, $(TARGET_DEVICE)),)
```

Comment:
Sorry, but the comma is wrong. Please change it to:

`ifneq ($(filter beyond0lte beyond1lte beyond2lte, $(TARGET_DEVICE)),)`

Reply:
Alright!